modules/ROOT/pages/api-led-design.adoc

```diff
@@ -0,0 +1,159 @@
+and properly structured request can access. 
+
+An API specification details the functional and expected behavior of an API, 
+as well as the fundamental design philosophy and supported data types.
+It contains both documentation and API definitions 
+to create a contract that humans and software can read. 
+
+MuleSoft provides tools that make it easy to create an API specification that you can then share with your team or publicly.
+The API specification increases adoption and speeds up project completion. 
+
+== Step 1: Explore Existing API Specifications
+If you look at existing API specifications before writing your own, you can learn how other people have approached similar problems. You can also check whether an asset with the same objectives is not already in development, and reuse it if appropriate.
+
+Looking for an API specification that already does what you need is easy:
+
+* Look at the public Exchange. You’ll see some of the most popular API specifications, connectors, and other assets displayed.
++
+link:https://www.mulesoft.com/exchange["Take me there^", role="button-primary"]
+ +
+Select *REST APIs* from the *Type* dropdown to display only REST API specifications. Click on any specification to see the data types and the HTTP requests defined for the API. 
++
+* To look at just the items in your organization (account) Exchange, 
+log in to your Anypoint Platform account (your account).
++
+link:https://anypoint.mulesoft.com/login["Take me there^", role="button-primary"] 
+
+== Step 2: Create Your API Specification
+In link:https://anypoint.mulesoft.com/login/["Anypoint Platform^", role="button-primary"]:
+
+. Click the blue button under Design Center to open the API Designer.
+. Click *Create* to open the API Designer editor.
+. Click *Create API specification*.
+. Enter Hello-World for *Name* and leave the other default values.
+. Click *Create Specification*.
+. The API Designer editor displays starter text. Replace it with RAML that specifies the API:
+
+[source,raml]
+----
+#%RAML 1.0
+title: Hello, World
+version: v1
+description: A greeting for the world
+
+securitySchemes:
+ basic:
+   description: |
+     This API supports Basic Authentication.
+   type: Basic Authentication
+ 
+securedBy: [basic]
+
+types:
+ greeting: # <2>
+   properties:
+     todays-greeting: string
+
+/greeting:
+ /{todays-greeting}:
+     uriParameters:
+       todays-greeting: string
+     get: # <1>
+       responses: # <3>
+         200:
+           body:
+             application/json:
+               type: greeting
+         404: # <4>
+           body:
+             application/json:
+               properties:
+                 message: string
+               example: |
+                 {
+                   "message" : "Greeting not found"
+                 }
+----
+<1> A single HTTP request, GET
+<2> A single `dataType`, `greeting`, with a single property, `todays-greeting` (today’s greeting message). 
```

Comment:
```suggestion
<2> A single `dataType` value, `greeting`, with a single property, `todays-greeting` (today’s greeting message)
```
OR
```suggestion
<2> A single data type, `greeting`, with a single property, `todays-greeting` (today’s greeting message). 
```
Not sure we need "(today’s greeting message)."

Reply:
doen